ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -170,17 +187,65 @@ void RSkComponentImage::processImageData(const char* path, char* response, int s
     }
     findImageData = SkImage::MakeFromEncoded(data);
```

Comment:
can we rename this variablename  "findImageData" . usage of "find" is mostly used for functions..probably can be names as remoteImageData or a relevant name